test/Quake/dependency-if-bug-lowering.qke

```diff
@@ -0,0 +1,49 @@
+// ========================================================================== //
```

Comment:
nit: The `-` in the filename should be `_` for consistency with the rest of the tests.

The filenames ought to follow a snake case convention, with a `-<n>` suffix for several variant tests of the same/similar thing.